### ISSUE TITLE
Edit APK without install...

### ISSUE THREAD

**Issue by Rstment25** [apk-editor/APK-Explorer-Editor#26]
Why is install the only option when you go into apk tab? Some apps are impossible to install - can only be dropped into /system folder so not being able to edit those without installing is a major pain in the ass ( Some apps you don't wanna install either without opening first and checking... Some might even need edit first to even be able to install - think unsupported android version although very rare nowadays - But your app does not allow any sort of edit without installing app on device )

It offers re signing tho so surely it can also offer full edit ? 

Can you add ability to do edit on selected apk file in a future release?

Thanks!

**Comment by ImaDoor2785** [comment]
yes this right here ive been looking for an apk editor that can do this and works on Android 13 but i haven't found any
